src/Util.hs

```diff
@@ -59,8 +59,10 @@ gitCheckIgnore files = do
   where
     join_ = intercalate "\0"
     split = map T.unpack . T.split (== '\0') . T.pack
+    notGitWarning = Just (Cyan, "warning: not a git repository - .gitignore support not available\n")
     feedback err
-      | err == "fatal: not a git repository (or any of the parent directories): .git\n" = Just (Cyan, "warning: not a git repository - .gitignore support not available\n")
+      | err == "fatal: not a git repository (or any of the parent directories): .git\n" = notGitWarning
+      | "fatal: not a git repository (or any parent up to mount point " `isPrefixOf` err = notGitWarning
```

Comment:
I would like to see the complete error message somewhere.  This is what I suggest:

1. Move `feedback` to the top level and rename it to `gitCheckIgnoreFeedback`
2. Add a test that passes the complete error message to `gitCheckIgnoreFeedback`